lib/db.js

```diff
@@ -434,6 +434,7 @@ module.exports = function (
             return bufferize({
               id: item.id,
               sessionToken: item.sessionTokenId,
+              lastConnectedAt: item.lastAccessTime,
```

Comment:
Fwiw I'm happy to change it, aesthetics are important too. What would you prefer it to be?